test/run-e2e-tests.sh

```diff
@@ -25,9 +25,9 @@ SKIP_INSTALL=${SKIP_INSTALL:-}
 if [ -n "$KIND_E2E" ]; then
     # If we did not set SKIP_INSTALL
     if [ -z "$SKIP_INSTALL" ]; then
-        K8S_VERSION=${KUBERNETES_VERSION:-v1.27.0}
+        K8S_VERSION=${KUBERNETES_VERSION:-v1.28.0}
         curl -Lo kubectl https://dl.k8s.io/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/
-        wget https://github.com/kubernetes-sigs/kind/releases/download/v0.18.0/kind-linux-amd64
+        wget https://github.com/kubernetes-sigs/kind/releases/download/v0.19.0/kind-linux-amd64
```

Comment:
shouldn't this be a blocker for us to migrate to 1.28?
i.e. if we cannot run our e2e on 1.28 nodes, then we don't get a 100% guarantee

Reply:
kind v0.20.0 can not start the cluster successfully. see here: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_descheduler/1216/pull-descheduler-test-e2e-k8s-master-1-26/1692035879192760320/build-log.txt
it's not descheduler issue.
```
[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory "/etc/kubernetes/manifests". This can take up to 4m0s
I0817 04:50:14.324118     150 round_trippers.go:553] GET https://kind-control-plane:6443/healthz?timeout=10s  in 0 milliseconds
```

Reply:
Thanks @JaneLiuL I understand that, but can we confidently upgrade to 1.28 if we do not run our e2e on 1.28 nodes?

Reply:
x-ref: https://kubernetes.slack.com/archives/CCK68P2Q2/p1692626226992449

descheduler was migrated to eks in PR: https://github.com/kubernetes/test-infra/pull/30433
and it seems to have resolved our kind/cgroup-v2 issues